src/snek5000_cbox/cbox.usr.f

```diff
@@ -279,7 +279,7 @@ subroutine usrdat2
 
       stretch_y = abs(UPARAM(4))
 
-      if (stretch_x.ne.0.0) then
+      if (stretch_y.ne.0.0) then
```

Comment:
I don't understand why it is still there. This PR should be based on a version for which we already have `stretch_y.ne.0.0`.